### PR TITLE
add cookie-banner

### LINKED ISSUE #32: Add cookie warning
I think we're required to show a cookie warning on the website actually 😬

https://www.cookielaw.org/the-cookie-law/

There might be an external service we can use rather than implement it ourselves.

### PR DESCRIPTION
**Pull request by trabaldini**
closes #32

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -486,6 +486,38 @@ i:hover {
...
+
+.cookie-button {
+  display: inline-block;
+  border-width: 2px;
+  border-style: solid;
+  padding: 5px 50px;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #e54e22;
```
Comment (marcoow) [inline css/main.scss]: This looks strange: 
bildschirmfoto 2018-01-24 um 15 12 36

**Review by marcoow — COMMENTED** [review]
Nice but the hover state of the _"OK"_ button looks strange.

**Comment by trabaldini** [comment]
Is white better?

**Comment by marcoow** [comment]
Yes, I think this looks much better

**Review by marcoow — APPROVED** [review]

**Pull request merged**